pyiso/nspower.py

```diff
@@ -0,0 +1,275 @@
+import pandas
+import pytz
+from datetime import timedelta, datetime
+from pyiso import LOGGER
+from pyiso.base import BaseClient
+
+
+class NSPowerClient(BaseClient):
+    """
+    Nova Scotia Power (Canada) has a public, informational page titled "Today's Power" which is backed by several
+    useful JSON data sources.
+
+    https://www.nspower.ca/en/home/about-us/todayspower.aspx
+
+    This client provides:
+        * historical loads up to 24 hours in the past and load forecasts up to 24 hours into the future.
+        * historical generation mix up to 24 hours in the past through to the start of the current hour.
+    """
+    NAME = 'NSP'
+    TZ_NAME = 'Canada/Atlantic'
+
+    # LM6000s are combined cycle gas turbines. I don't know if the value being listed seperately represents just the
+    # condensing steam generator (i.e. Tuft Cove 6) or the entire combined cycle system of two natural gas generators
+    # (Tuft Cove 4 & 5) plus the condensing steam generator (Tuft Cove 6).
+    # See http://www.novascotia.ca/nse/ea/tuftscove6/NSPI-TuftsCove6-Registration.pdf
+    #
+    # CTs are diesel (oil) combustion turbines (I think).
+    # See http://www.nspower.ca/en/home/about-us/how-we-make-electricity/thermal-electricity/oil-facilities.aspx
+    #
+    # "HFO/Natural Gas" is Heavy Fuel Oil/Natural Gas
+    fuels = {
+        'Solid Fuel': 'coal',
+        'HFO/Natural Gas': 'natgas',
+        'CT\'s': 'oil',
+        'LM 6000\'s': 'thermal',
+        'Biomass': 'biomass',
+        'Hydro': 'hydro',
+        'Wind': 'wind',
+        'Imports': 'other'
```

Comment:
@xinyue-luna, should imports be reflected as `other` in the generation mix? They're part of the electricity mix required to fulfill the system load, but it's not generation. There is no export data provided to derive `get_trade(...)` so it can only be reflected as part of the `get_generation(...)` mix.